src/ucp/core/ucp_context.c

```diff
@@ -214,6 +214,25 @@ void ucp_config_print(const ucp_config_t *config, FILE *stream,
                                  print_flags);
 }
 
+static int ucp_str_array_search_aux(const char **array, unsigned length,
```

Comment:
pls, add description of return value and rename `length` to something like `alength` to be more clear lenght of what.
Also, if we introduce some special format then maybe to rename the function to something like `ucp_tls_array_format_search` with extra argument what should it search? then it maybe joined with `ucp_str_array_search`